dashboard/lib/logic/qualified_task.dart

```diff
@@ -14,12 +14,14 @@ class StageName {
   static const String cocoon = 'cocoon';
   static const String legacyLuci = 'chromebot';
   static const String luci = 'luci';
+  static const String frob = 'frob';
```

Comment:
Do we expect to have more tests other than `google test` in frob? Team is more familiar with the term google-test rather than frob.

Reply:
thanks for the advice! Shall I go ahead and change every frob naming to be google-test instead?

Reply:
Google test sgtm